doc/sphinx/source/libCEEDdev.rst

```diff
@@ -42,6 +42,53 @@ are purely implicit -- one just indexes the same array using the
 appropriate convention.
 
 
+Internal Layouts
+----------------------------------------
+
+Ceed backends are free to use any **E-vector** and **Q-vector** data layout, to include never fully forming these vectors, so long as the backend passes the ``t5**`` series tests and all examples.
+There are several common layouts for **L-vectors**, **E-vectors**, and **Q-vectors**, detailed below:
+
+* **L-vector** layouts
+
+  * **L-vectors** described by a :ref:`CeedElemRestriction` have a layout described by the ``offsets`` array and ``compstride`` parameter.
+    Data for node ``i``, component ``j``, element ``k`` can be found in the **L-vector** at index ``offsets[i + j*elemsize] + k*compstride``.
+
```

Comment:
I don't think this is correct.  Per the comments in `ceed-elemrestriction.c`, 

```
offsets    Array of shape [@a nelem, @a elemsize]. Row i holds the
                      ordered list of the offsets (into the input CeedVector)
                      for the unknowns corresponding to element i, where
                      0 <= i < @a nelem. All offsets must be in the range
                      [0, @a lsize - 1].
```
So this should be `offsets[i + k*elemsize] + j*compstride`, no?

Reply:
Crap. Good catch, bad copy-paste from me